spec/public-api.yaml

```diff
@@ -132,6 +132,13 @@ paths:
           type: string
         description: Choose a detector by its ID.
         required: true
+      - in: query 
+        name: patience_time 
+        schema: 
+          type: number 
+          format: float 
+        description: How long to wait for a confident response. 
+        required: true 
```

Comment:
I'm thinking about whether we want this parameter to be required or not. My initial thought is to not make it required, so that client code written against the old API can still be valid. But there could be different interpretations of `patience_time=None` (either "return immediately", or "use the default of 30 seconds").

Anyway, API changes are important enough that we should get a second opinion from @tyler-romero / @robotrapta.

Reply:
Agreed this should be an optional parameter here.  If not supplied to the SDK method, we shouldn't send it to the API.  So the API also needs to have it be optional.